docs/components/histogram.md

```diff
@@ -13,13 +13,13 @@ Talk about the Histogram
 
 #### **data** (array)
 
-Value that will be shown in the widget. Each category should include a `name` and a `value`. For example:
+Value that will be shown in the widget. Each bin should include `start`, `end` and `value` keys. For example:
 
 ```
 [
-  { name: 'A New Hope', value: 359029623 },
-  { name: 'The Empire Strikes Back', value: 236513856 },
-  { name: 'Return of the Jedi', value: 204338075 },
+  { start: 8, end: 48, value: 6318 },
+  { start: 49, end: 88, value: 5922 },
```

Comment:
Limits are inclusive. I mean, one bin has `end: 48` and the next one `starts: 48`.

Reply:
Since we're relying in the code on having the input data sorted, let's reflect that in the documentation.

`Bin data should be provided sorted and complete:
  - Sorted: bin order must respect data order. `bin[0]` should start before bin[1] and so on.
  - Complete: if a bin doesn't have any value, it must be provided anyway with the proper `start` and `end` properties and a `value: 0`.

Reply:
Done!